modules/trusted-firmware-m/interface/interface.c

```diff
@@ -45,53 +49,18 @@ int32_t tfm_ns_interface_dispatch(veneer_fn fn,
 	k_sched_lock();
 #endif
 
-#if defined(CONFIG_ARM_NONSECURE_PREEMPTIBLE_SECURE_CALLS) && defined(CONFIG_FPU_SHARING)
-	uint32_t fp_ctx_caller_saved[16];
-	uint32_t fp_ctx_callee_saved[16];
-	uint32_t fp_ctx_FPSCR;
+	uint32_t context_buffer[33];
 	bool context_saved = false;
```

Comment:
@oyvindronningstad i think this variable needs to be inside the #ifdef as well, to suppress the unused variable error